src/pudl/metadata/resources/eiaaeo.py

```diff
@@ -0,0 +1,524 @@
+            "sell some power to the grid. Solar photovoltaic capacity portion of "
+            "Renewable Sources in gigawatts direct current; other technologies "
+            "in gigawatts alternating current."
+        ),
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+                "gross_generation_mwh",
+                "sales_mwh",
+                "generation_for_own_use_mwh",
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__yearly_projection_electricity_sales": {
+        "description": "Projected electricity sales.",
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+                "customer_class",
+                "sales_mwh",
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+                "customer_class",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__yearly_projection_net_energy_for_load": {
+        "description": "Projected net energy for load.",
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+                "generation_source_or_sink_type_aeo",
+                "net_energy_for_load_mwh",
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+                "generation_source_or_sink_type_aeo",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__yearly_projection_end_use_prices_by_sector": {
+        "description": "Projected electricity cost to the end user by sector.",
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+                "customer_class",
+                "nominal_price_per_mwh",
```

Comment:
i wonder if this guy should just be `price_per_mwh`? bc i think by default the $s across our tables are nominal and the real-ness is the exception. that's a lil weirder in the context of projections so i think there is a fine argument for keeping nominal. (also my always suggestion to move the "what" to the front of the name and qualifiers later for easier finding/sorting despite being slightly less colloquial:`price_nominal_per_mwh` and `price_real_per_mwh`)